.ci/azure-pipelines/env.yml

```
@@ -58,6 +58,13 @@ jobs:
         # nvidia-cuda docker image has not been released for 20.04 yet
         USE_CUDA: ""
         tag: 20.04
+      Ubuntu 20.10:
+        CUDA_VERSION: 11
+        UBUNTU_DISTRO: 20.10
+        VTK_VERSION: 8
```

Apparently, even 20.10 doesn't have VTK 8 in apt.
```suggestion
        VTK_VERSION: 7
```